kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -89,6 +97,11 @@ window.LoginView = BaseView.extend({
     handle_login: function(response) {
         if (response.status == 200) {
             this.trigger("login_success");
```

Comment:
> I don't know how this could possibly result in an unsafe expectation.

It doesn't right now, but my point is that it could in the future if someone decides to add another listener. Since this _could_ trigger a page load (yet _may_ not), any listener will have to finish what it's doing in a hurry or potentially be cut short. If all you wish to do is close the modal, why not just call `close_modal` directly?